src/View/layout.html.twig

```diff
@@ -34,7 +25,6 @@
 
```

Comment:
no content block left!

Reply:
it's normal because left is a other feature

Reply:
it's normal , the block left is other feature